Framework/Geometry/src/Math/Acomp.cpp

```diff
@@ -214,8 +214,8 @@ Complementary subtraction is by making A-B == A*B'
 
   // Have DNF parts and will return the same...
   // Sort the components of the list
-  std::for_each(Fparts.begin(), Fparts.end(), std::mem_fun_ref(&Acomp::Sort));
-  std::for_each(Gparts.begin(), Gparts.end(), std::mem_fun_ref(&Acomp::Sort));
+  std::for_each(Fparts.begin(), Fparts.end(), std::mem_fn(&Acomp::Sort));
```

Comment:
Same with `sort` and a lambda

Reply:
So this is not actually sorting `Fparts`/`Gparts` (that is done below). It's calling `Acomp::Sort` on each entry in `Fparts`/`Gparts`. I'm not sure a lambda clarifies this much?